packages/react/.storybook/addon-fathom/register.js

```diff
@@ -0,0 +1,27 @@
+import { window } from 'global';
+import { addons } from '@storybook/addons';
+import { STORY_CHANGED } from '@storybook/core-events';
+
+addons.register('fathom', api => {
+  (function(f, a, t, h, o, m) {
```

Comment:
Should we wrap this or the set for siteId in a block for production deploys:

```js
if (process.env.NODE_ENV === 'production') {
  // start tracking
}
```

So development stats don't get logged? It could easily be that we're doing this after testing is done! I just wanted to check.

Reply:
Yup, will definitely make sure we only track production stats once we get it all sorted!